corehq/apps/repeaters/repeater_generators.py

```diff
@@ -315,8 +315,6 @@ def get_test_payload(self, domain):
 
 class UserPayloadGenerator(BasePayloadGenerator):
 
-    deprecated_format_names = ('user_json',)
```

Comment:
Just checking but I'm assuming what you said about `LocationPayloadGenerator` also applies to `UserPayloadGenerator`? I.e. those repeaters were both only recently created and have no live usages.